### DISCUSSION TITLE
Observational Talks Q&A

### DISCUSSION THREAD

**Discussion by gavinny** [NASA-GISS/freshwater-forcing-workshop#10] [Talks Q&A]
Use this thread to ask questions about the observational talks in the first section.

**Comment by xylar** [comment]
In his talk, Ben Davison (sorry, my Google stalking wasn't able to find your GitHub handle) presented time series of melt rates derived in various ways from satellite observations.  He points out that the these products don't agree on the magnitude or timing of sub-decadal variability.  I wanted to point out as well that ApRES (autonomous phase-sensitive radar) measurements of melt rates also typically show less temporal variability than any of the satellite-derived products.  A postdoc, Irena Vankova, that I work with knows more by far about this than I do.  @irenavankova, would you care to comment?

To me, this suggests that we don't want to rely too heavily on the satellite-derived products to add temporal variability on timescales shorter than decadal to whatever freshwater anomaly protocol we might come up with.

**Comment by Recalculate** [comment]
In his talk, @therealpaulholland mentioned that most CMIP models will presently input any increase in precipitation over ice sheets immediately back into the ocean as a fw runoff, rather than first storing it as snow/ice for 100-1000s of years as they should.    My question is how large is this spurious precipitation run off increase over the historical period, and how close is it to offsetting what we should be inputting from enhanced ice sheet melt?

**Reply by xylar** [reply]
I think that was covered in Matt England's talk at around timestamp 6:35.

![Screenshot from 2024-02-12 13-05-18](https://github.com/NASA-GISS/freshwater-forcing-workshop/assets/4179064/48615154-8c83-4aa2-848e-76154d1f4e59)

From that slide, it looks like maybe the main problem isn't over the historical but using that technique for projections.

But as @therealpallholland pointed out, that problem isn't just the magnitude of the error but the incorrect seasonality, incorrect geographic distribution and incorrect depth at which FWF are input.

**Reply by therealpaulholland** [reply]
The same conversation is also going on in the other discussion, sorry!  I wonder how representative the purple bars shown here are of the 'best guess' for SSP585.  That is something I guess we will find out later on in the meeting.
